docs/design/da-pki-revocation-design.md

```diff
@@ -0,0 +1,213 @@
+
+2. Revocation of a DAC 
+
+   A Vendor with DCL write privilege can submit a transaction to publish the location of the CRL distribution point for a PAI associated with the Vendor ID. This information can be updated by the Vendor.
+
+
+## Revocation Distribution Point Schema
+The schema is the same as in `PR6360`.
+However, the validation logic is proposed to be a bit different (see [Validation Logic](#validation-logic) and [Divergence](#divergence-from-pr6360)). 
+
+
+| Name                 | Type    | Constraint          | Mutable | Conformance |
+|:---------------------|:--------|:--------------------|:--------|-------------|
+| VendorID             | uint16  | all                 | No      | M           |
+| ProductID            | uint16* | all                 | No      | O (desc)    |
+| IsPAA                | bool    | all                 | No      | M           |
+| Label                | string  | max 64              | No      | M           |
+| CRLSignerCertificate | string  | max 2048            | No      | M           |
+| IssuerSubjectKeyID   | string  | max 64              | No      | M           |
+| DataUrl              | string  | max 256             | Yes     | M           |
+| DataFileSize         | uint64  | desc                | Yes     | O           |
+| DataDigest           | string  | max 128             | Yes     | O (desc)    |
+| DataDigestType       | uint32  | all                 | Yes     | O (desc)    |
+| RevocationType       | uint32  | all                 | No      | M           |
+
+*: `PR6360` defines PID as `uint32` which is incorrect. PID must be `uint16` as in other places.
+
+## Divergence from PR6360
+1. No validation of `DataUrl` content.
+   If the content is invalid, it can be edited/removed by either owner Vendor or Trustees via Update and Delete commands.
+
+   Such validation requires resolving an external URL, and this is a non-deterministic operation:
+   some nodes may resolve the URL successfully, but some nodes may not resolve it or be redirected to a different content.
+   Non-deterministic results of validation may cause a situation that on some nodes the transaction is considered as valid,
+   and on some nodes as invalid. That will break the consensus protocol and DCL won't be able to process write requests anymore.
+   An alternative option is to pass the whole content of `DataUrl` to the transaction, but it doesn't make a lot of sense because
+    1. the content can be quite big (1 MB)
+    2. there will be no guarantee that `DataUrl` actually points to the same content as passed
+
+2. Validation of `CRLSignerCertificate`. If `CRLSignerCertificate` is a PAA, then it must be present on DCL (there must be a PAA on DCL with the same pem value).
+   If `CRLSignerCertificate` is a PAI, then it must be chained back to a valid PAA present on the ledger.
```

Comment:
- Chaining should be validated via:
  - Query for a PAA where `Subject == CRLSignerCertificate.Issuer` and `SubjectKeyID == CRLSignerCertificate.AuthorityKeyId"
  - Build certification path with both elements, verify path.

Reply:
Yes, this is exactly what's meant here. I will add the clarification.